### ISSUE TITLE
Displaying sidecar log on dashboard

### ISSUE THREAD

**Issue by wingyplus** [tektoncd/dashboard#1833]
# Is your feature request related to a problem ?

Currently, when sidecar has a problem and terminate abnormally, it'll not know until I use kubectl to log the pipeline and see what's going on with the sidecar. For example, I've a docker task set up which use dockerd as a sidecar, when I setup sidecar incorrectly (such as setting incorrect flag), dashboard will display something like this screenshot below:

![Screen Shot 2563-11-17 at 23 15 41](https://user-images.githubusercontent.com/484530/99416042-02120500-292b-11eb-9011-1a81f53db5f1.png)

This task fail correctly because the dockerd (sidecar) doesn't start and it fail. But it's hard to know what's going on. :)

# Describe the solution you'd like

The solution might be add a new sidecar task under the step. I've come from Drone CI, it's done like this:

![Screen Shot 2563-11-17 at 23 11 28](https://user-images.githubusercontent.com/484530/99415502-713b2980-292a-11eb-9316-28c746e229e0.png)

From screenshot above, MySQL Database is the sidecar for this pipeline. The bad on this design is that we didn't know on UI which one is sidecar. 

# Describe alternatives you've considered

Currently, it can be solved by inspect by using kubectl like this:

```
$ kubectl logs  pod/pipeline-run-m8q6l-build-and-publish-dock-ndtxt -c sidecar-docker-daemon -n my_namespace
```

**Comment by AlanGreene** [comment]
Thanks for the feature request @wingyplus. This is already being tracked in https://github.com/tektoncd/dashboard/issues/1833 so I'll close this one as a duplicate.

**Comment by wingyplus** [comment]
@AlanGreene your mentioned is this issue. The correct one is #972

**Comment by AlanGreene** [comment]
Yes that's the correct one, I updated my comment just after I posted it. Sorry for the confusion.

**Comment by wingyplus** [comment]
@AlanGreene thanks for your inform.
